styles/style.css

```diff
@@ -0,0 +1,448 @@
+  transition: all 1s;
+}
+
+.search__bar { -webkit-appearance: none;}
+textarea:focus, input:focus, button:focus { outline: none; }
+
+.product__radio { display: none; }
+
+figure { margin: unset; }
+
+.nav_caps, .header_caps, .product__header_caps, .stories__header_caps, .story__link_caps, .photos__header_caps { text-transform: uppercase; }
+.nav_bold, .footer__header, .product__price {
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.nav {
+  display: grid;
+  grid: 4.5rem / auto auto;
+}
+
+.nav__items, .nav__links {
+  display: grid;
+  text-align: center;
+  grid-column: 1/-1;
+}
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__link_col, .nav__signin_col, .nav__cart_col, .nav__lang_col { height: 10vw; }
+
+.nav__home {
+  display: flex;
+  align-items: center;
+}
+.nav__logo_mob { width: 6.5rem; }
+.nav__logo_desk {
+  width: calc(6.45rem - 4.13vw);
+  min-width: 4.58em;
+}
+
+.nav__hamb {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-self: flex-end;
+}
+.nav__bar {
+  width: 2.56rem;
+  height: 0.3125rem;
+}
+.nav__bar_black { background: black; }
+
+.nav__amount {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  height: 1rem;
+  width: 1rem;
+  color: #fff;
+  background: #000;
+  border-radius: 50%;
+  margin: .3em;
+}
+
+.nav__checkbox_none, .nav__items_none { display: none; }
+.nav__checkbox:checked ~ .nav__items { display: grid; }
+.nav__home_desk { display: none; }
+
+.header {
+  height: 11.94rem;
+  display: grid;
+  grid-template-rows: 5.4em 6.3em;
+  /* background: url(../logos/rob-potter-422534-unsplash.png) no-repeat -7.5em/cover; */
+  background: url(../logos/rob-potter-422534-unsplash.png) no-repeat calc(50.18vw - 20.88em)/cover;
+}
+
+.header__title { grid-area: 2; }
+.header__link {
+  display: flex;
+  grid-area: 3;
+  justify-self: start;
+  align-self: flex-start;
+}
+.info__link { display: none; }
+
+.product {
+  display: grid;
+  height: 29.9em;
+  grid-template-rows: 2.8em min-content;
+}
+.product__header {
+  grid-area: 2;
+  text-align: center;
+  padding-right: .5em;
+}
+
+.product__items {
+  grid-area: 3;
+  display: flex;
+  justify-content: center;
+}
+.product__item {
+  display: grid;
+  height: 18.3em;
+  width: 22.24em;
+  margin-left: 1.2em;
+  position: relative;
+}
+.product__item:nth-child(2), .product__item:nth-child(3), .product__item:nth-child(4) { display: none; }
+
+.product__link {
+  display: grid;
+  grid-template-rows: 14.9em 1.6em;
+}
+
+.product__clrs {
+  position: absolute;
+  display: flex;
+  justify-content: space-between;
+  width: 6.2em;
+  justify-self: end;
+  align-self: end;
+  align-items: center;
+}
+
+.product__img {
+  width: 13.55em;
+  height: 12.8em;
+  justify-self: center;
+}
+
+.product__clr {
+  width: 1.25em;
+  height: 1.25em;
+  border-radius: 50%;
+}
+.product__clr_red {background: #B62E3C;}
+.product__clr_bl {background: #0E1211;}
```

Comment:
Codacy found an issue: [Expected single space after "{" of a single-line block (block-opening-brace-space-after)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887702)